cluster-autoscaler/cloudprovider/openstack/examples/os-autoscaler-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: os-autoscaler
+  namespace: kube-system
+  labels:
+    app: autoscaler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: autoscaler
+  template:
+    metadata:
+      namespace: kube-system
+      labels:
+        app: autoscaler
+    spec:
+      serviceAccountName: os-autoscaler-account
+      containers:
+        - name: os-autoscaler
+          image: thartland/os-autoscaler:v1.0
+          imagePullPolicy: Always
+          command:
+            - ./cluster-autoscaler
+            - --alsologtostderr
+            - --cluster-name=scaler-01
+            - --cloud-config=/config/cloud-config
+            - --cloud-provider=openstack
+            - --nodes=1:10:DefaultNodeGroup
+            - --scale-down-unneeded-time=30s
```

Comment:
Same as above.

Reply:
Done in https://github.com/kubernetes/autoscaler/pull/1690/commits/1e3f725274f112baed9c1608fca60fa9cd7f0daf.